README.md

```diff
@@ -108,6 +135,33 @@ ms3 = marginals(y₂′(Xp));
 μf′, σf′ = mean.(ms1), std.(ms1);
 μy₁′, σy₁′ = mean.(ms2), std.(ms2);
 μy₂′, σy₂′ = mean.(ms3), std.(ms3);
+
+# Instantiate plot and chose backend
+plotly();
+posterior_plot = plot();
+
+# Plot posteriors
+plot!(posterior_plot, y₁′(Xp); samples=S, sample_seriestype=:scatter, color=:red, label="")
```

Comment:
Add semi-colons here to suppress plotting intermediates?

Reply:
My bad, you are absolutely right :)